src/Microsoft.OData.Core/ODataMessageReaderSettings.cs

```diff
@@ -67,7 +67,7 @@ public ODataMessageReaderSettings(ODataVersion odataVersion)
         }
 
         /// <summary>
-        /// Gets or sets library compatibility version. Default value is <see cref="Microsoft.OData.ODataLibraryCompatibility.Latest"/>,
+        /// Gets or sets library compatibility version. Default value is <see cref="Microsoft.OData.ODataLibraryCompatibility.None"/>,

```

Comment:
did we change this in 8.x? why?

Reply:
There's no 'Latest' enum member for ODataLibraryCompatibility, And I noticed that in the constructor, 'None' is used for default value. I didn't see any changes within 8.x version for this part..

Reply:
I think that was changed in one of 7 version.

Reply:
found the [commit ](https://github.com/OData/odata.net/commit/fde7d8e0f02879799ac06a4de3dddc2613d054af) that updated this. Thanks for catching the comment!